lib/Onyx.js

```diff
@@ -639,8 +680,9 @@ function keyChanged(key, data, canUpdateSubscriber, notifyRegularSubscibers = tr
  * @param {String} [mapping.selector]
  * @param {*|null} val
  * @param {String} matchedKey
```

Comment:
I can see below the `matchedKey` can be `undefined`, should we update the doc here?